core/src/repair/cluster_slot_state_verifier.rs

```diff
@@ -456,8 +455,8 @@ fn check_epoch_slots_hash_against_bank_status(
             assert!(is_popular_pruned);
             // The cluster sample found the troublesome slot which caused this fork to be pruned
             warn!(
-                "EpochSlots sample returned slot {slot} with hash {epoch_slots_frozen_hash}, but \
-                 we have pruned it due to incorrect ancestry"
+                "EpochSlots sample returned slot {slot} with hash {epoch_slots_frozen_hash}, \
+                 but we have pruned it due to incorrect ancestry"
```

Comment:
Out of curiosity, did `fmt` make this change or did you ?

Reply:
I did